src/app/resize/resize.component.ts

```diff
@@ -0,0 +1,72 @@
+import {Component, ElementRef, EventEmitter, Input, Output} from "@angular/core";
+
+@Component({
+	selector: 'resize',
+	templateUrl: `
+		<div [class.overlay]="isExpanded"></div>
+		<div
+			class="handle"
+			[class.is-expanded]="isExpanded">
+		</div>
+		`,
+	styles: [`
+	.handle {
+		position: absolute;
+		right: 0;
+		top: 0;
+		height: 100%;
+		width: 15px;
+		border-right: 1px dashed rgba(128, 128, 128, 0.5);
```

Comment:
It feels a bit weird that handle in only on the right side of the line without any visual indication of that. 
Maybe add a nested div to get the border in the middle of the handle?